### PR TITLE
Added conda install info to readme

### LINKED ISSUE #4: Conda-forge release
We'll need to get automation up to release to conda-forge from PyPI. 

Depends on #3.

### PR DESCRIPTION
**Pull request by sugatoray**
Added conda install instruction to `README.md`.

This is representative of the work done to close #4.

### CODE REVIEW

**Comment by sugatoray** [comment]
cc: @ericmjl The package on PyPI does not show any python version. For clarity, please see the comparison between `boubtlab` and `seqlike` on supported python version(s) from PyPI.

**For `doublab`**

<p>
  <a href="https://pypi.org/project/doubtlab/">
    PyPI - Supported Python Version
  </a>
</p>

**For `seqlike`**

<p>
  <a href="https://pypi.org/project/seqlike/">
    PyPI - Supported Python Version
  </a>
</p>

**Comment by ericmjl** [comment]
@sugatoray thanks for handling this! I think it's reasonable for us to follow [NEP29](https://numpy.org/neps/nep-0029-deprecation_policy.html) and support essentially the "last three versions" of Python. The classifiers you've proposed look like the right ones, please go ahead and add them in :smile:. 

Meanwhile, on the failing test, I'm not sure what's going on, but I'll be debugging on your branch to see what's going on.

**Comment by ericmjl** [comment]
@sugatoray please merge in the latest main branch; it'll contain a fix for the unit tests.

**Comment by sugatoray** [comment]
@ericmjl Done. Also, I marked the setup.py version as `1.1.7.dev1`. This will allow you to push new changes on an increamental basis to PyPI, but as a pre-release (marked by PyPI default, because of the `.dev#` part).

So, typically, you could be pushing multiple `v1.1.7.dev1`, `v1.1.7.dev2`, `v1.1.7.dev3`, etc. `PRE-RELEASE`'s before finally shipping out `v1.1.7` to PyPI.

## An opinionated suggestion

Once a certain (non-pre-release) version (`X.Y.Z`) is released on PyPI, I would suggest you to immediately update the setup.py version to `X.Y.Z+1.dev1`. This makes it clear for anyone that the current changes will be part of this pre-release version.

File: setup.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def download_mafft(kind="deb") -> Path:
 
 setup(
     name="seqlike",
-    version="1.1.6",
+    version="1.1.7.dev1",
```
Comment (ericmjl) [inline setup.py:113]:
```suggestion
    version="1.1.6",
```

We have a GitHub Action for cutting releases that depend on having a sync'd up version number with the appropriate configuration files. Therefore, I'd like to request that we change this back to 1.1.6.

Also, while I appreciate the sentiment for having dev releases, until I learn how to configure bump2version to include dev as part of the version (and thus update the appropriate GitHub action that we have), we're probably going to stick with major/minor/patch as our convention for now. That can be revisited in the future, though.

**Review by ericmjl — CHANGES_REQUESTED** [review]
Just one last change requested, @sugatoray, to revert the version number. I've explained why in the appropriate comment.

File: setup.py (hunk 1 above)
Comment (sugatoray) [reply setup.py:129]: Thank you, for explaining.

File: setup.py [hunk 2]
```diff
@@ -108,27 +108,49 @@ def download_mafft(kind="deb") -> Path:
...
-        "python-codon-tables",
-    ],
+    long_description=README,
+    long_description_content_type="text/markdown",
+    url="https://github.com/modernatx/seqlike",
+    project_urls={
+        "Documentation": "https://modernatx.github.io/seqlike",
+        "Source Code": "https://github.com/modernatx/seqlike",
+        "Issue Tracker": "https://github.com/modernatx/seqlike/issues",
+    },
```
Comment (sugatoray) [inline setup.py:139]: @ericmjl I also added this metadata. Please let me know, if this looks okay.

```suggestion
    long_description=README,
    long_description_content_type="text/markdown",
    url="https://github.com/modernatx/seqlike",
    project_urls={
        "Documentation": "https://modernatx.github.io/seqlike",
        "Source Code": "https://github.com/modernatx/seqlike",
        "Issue Tracker": "https://github.com/modernatx/seqlike/issues",
    },
```

Comment (ericmjl) [reply setup.py:139]: Amazing add, thank you very much!

**Review by ericmjl — APPROVED** [review]

**Comment by ericmjl** [comment]
Thank you, @sugatoray!

**Comment by sugatoray** [comment]
@ericmjl I just added you to the seqlike-feedstock as a co-maintainer.

- https://github.com/conda-forge/seqlike-feedstock/pull/2

**Pull request merged**